deliverables/README.md

```diff
@@ -31,38 +37,98 @@ What is the role of Javascript in a web page?
 
 #### meta
 Why do we separate HTML and CSS?  
+You seperate the HTML and CSS file so that you can make changes to the content on one file easily and then make changes to the style of the content efficiently with just one file instead of having to change each part of the html file.  
```

Comment:
Code maintainability is important, but we also separate them because they have different roles - HTML is structure, CSS is style.